[2023-03-07 12:07:58,789] {processor.py:153} INFO - Started process (PID=40471) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:07:58,789] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 12:07:58,790] {logging_mixin.py:115} INFO - [2023-03-07 12:07:58,790] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:07:58,801] {logging_mixin.py:115} INFO - [2023-03-07 12:07:58,801] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 12:07:58,801] {logging_mixin.py:115} INFO - [2023-03-07 12:07:58,801] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 12:07:58,801] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:07:58,807] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2023-03-07 12:08:36,363] {processor.py:153} INFO - Started process (PID=40762) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:08:36,364] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 12:08:36,365] {logging_mixin.py:115} INFO - [2023-03-07 12:08:36,365] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:08:36,396] {logging_mixin.py:115} INFO - [2023-03-07 12:08:36,396] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 12:08:36,396] {logging_mixin.py:115} INFO - [2023-03-07 12:08:36,396] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 12:08:36,396] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:08:36,405] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.050 seconds
[2023-03-07 12:09:09,641] {processor.py:153} INFO - Started process (PID=40850) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:09:09,642] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 12:09:09,642] {logging_mixin.py:115} INFO - [2023-03-07 12:09:09,642] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:09:09,654] {logging_mixin.py:115} INFO - [2023-03-07 12:09:09,654] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 12:09:09,655] {logging_mixin.py:115} INFO - [2023-03-07 12:09:09,655] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 12:09:09,655] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:09:09,661] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-03-07 12:09:40,080] {processor.py:153} INFO - Started process (PID=40937) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:09:40,082] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 12:09:40,083] {logging_mixin.py:115} INFO - [2023-03-07 12:09:40,083] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:09:40,113] {logging_mixin.py:115} INFO - [2023-03-07 12:09:40,113] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 12:09:40,113] {logging_mixin.py:115} INFO - [2023-03-07 12:09:40,113] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 12:09:40,114] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:09:40,121] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2023-03-07 12:10:12,750] {processor.py:153} INFO - Started process (PID=41009) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:10:12,751] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 12:10:12,752] {logging_mixin.py:115} INFO - [2023-03-07 12:10:12,752] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:10:12,771] {logging_mixin.py:115} INFO - [2023-03-07 12:10:12,771] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 12:10:12,771] {logging_mixin.py:115} INFO - [2023-03-07 12:10:12,771] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 12:10:12,771] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:10:12,776] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-03-07 12:10:44,747] {processor.py:153} INFO - Started process (PID=41091) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:10:44,749] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 12:10:44,750] {logging_mixin.py:115} INFO - [2023-03-07 12:10:44,750] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:10:44,781] {logging_mixin.py:115} INFO - [2023-03-07 12:10:44,781] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 12:10:44,781] {logging_mixin.py:115} INFO - [2023-03-07 12:10:44,781] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 12:10:44,781] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:10:44,788] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.047 seconds
[2023-03-07 12:11:19,514] {processor.py:153} INFO - Started process (PID=41187) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:11:19,515] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 12:11:19,515] {logging_mixin.py:115} INFO - [2023-03-07 12:11:19,515] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:11:19,527] {logging_mixin.py:115} INFO - [2023-03-07 12:11:19,526] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 12:11:19,527] {logging_mixin.py:115} INFO - [2023-03-07 12:11:19,527] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 12:11:19,527] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:11:19,532] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2023-03-07 12:11:37,615] {processor.py:153} INFO - Started process (PID=41293) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:11:37,615] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 12:11:37,615] {logging_mixin.py:115} INFO - [2023-03-07 12:11:37,615] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:11:37,625] {logging_mixin.py:115} INFO - [2023-03-07 12:11:37,625] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 12:11:37,625] {logging_mixin.py:115} INFO - [2023-03-07 12:11:37,625] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 12:11:37,626] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:11:37,630] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2023-03-07 12:12:10,304] {processor.py:153} INFO - Started process (PID=41413) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:12:10,305] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 12:12:10,306] {logging_mixin.py:115} INFO - [2023-03-07 12:12:10,306] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:12:10,319] {logging_mixin.py:115} INFO - [2023-03-07 12:12:10,318] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 12:12:10,319] {logging_mixin.py:115} INFO - [2023-03-07 12:12:10,319] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 12:12:10,319] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:12:10,327] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-03-07 12:12:40,532] {processor.py:153} INFO - Started process (PID=41505) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:12:40,533] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 12:12:40,535] {logging_mixin.py:115} INFO - [2023-03-07 12:12:40,535] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:12:40,563] {logging_mixin.py:115} INFO - [2023-03-07 12:12:40,562] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 12:12:40,563] {logging_mixin.py:115} INFO - [2023-03-07 12:12:40,563] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 12:12:40,563] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:12:40,571] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2023-03-07 12:13:14,704] {processor.py:153} INFO - Started process (PID=41593) to work on /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:13:14,705] {processor.py:641} INFO - Processing file /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-07 12:13:14,706] {logging_mixin.py:115} INFO - [2023-03-07 12:13:14,706] {dagbag.py:507} INFO - Filling up the DagBag from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:13:14,733] {logging_mixin.py:115} INFO - [2023-03-07 12:13:14,733] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-07 12:13:14,733] {logging_mixin.py:115} INFO - [2023-03-07 12:13:14,733] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-07 12:13:14,733] {processor.py:653} WARNING - No viable dags retrieved from /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-07 12:13:14,741] {processor.py:161} INFO - Processing /home/veri-pc/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
